src/site/content/en/payments/handling-optional-payment-information/index.md

```diff
@@ -200,13 +200,271 @@ object.
 Use the object to update the UI on the frontend. See [Reflect the updated
 payment details](#reflect-the-updated-payment-details).
 
+## Inform the merchant of a shipping address change {: #shipping-address-changes }
+
+Payment apps can provide the customer's shipping address to the merchant as part
+of the result of a payment transaction.
+
+This is useful for merchants because they can delegate the address collection to
+payment apps. And, because the address data will be provided in [the standard
+data format](https://w3c.github.io/payment-request/#dom-addressinit), the
+merchant can expect to receive shipping addresses in consistent structure.
```

Comment:
Can be rephrased as:

The merchant can expect to receive shipping addresses in the [standard data format](https://w3c.github.io/payment-request/#dom-addressinit).

Reply:
This will be repetitive and I think this is fine as is.